src/meshcat/servers/zmqserver.py

```diff
@@ -116,12 +117,20 @@ def f(port):
         else:
             self.zmq_socket, self.zmq_stream, self.zmq_url = self.setup_zmq(zmq_url)
 
+        protocol = "http:"
+        listen_kwargs = {}
+        if certfile is not None or keyfile is not None:
```

Comment:
Is there ever a case where you'd provide only the `certfile` but not the keyfile (or vice versa)? If not, we should throw an error here if one is missing. Or, alternatively, take them as a namedtuple so we can ensure that we're not missing one or the other.

Reply:
Done.